src/Congo.RazorPages/Pages/Products.cshtml.cs

```diff
@@ -18,15 +20,17 @@ public ProductsModel(IProductsService productsService)
             _productsService = productsService;
         }
 
-        public void OnGet()
+        public async Task<IActionResult> OnGet()
```

Comment:
```suggestion
        public async Task<IActionResult> OnGetAsync()
```

Not a requirement but can add the async keyword here since it will be ignored, same with the OnPost